Src/WitsmlExplorer.Frontend/components/ContentViews/MudLogsListView.tsx

```diff
@@ -1,17 +1,17 @@
 import React, { useContext, useEffect, useState } from "react";
 import NavigationContext from "../../contexts/navigationContext";
-import MudLogObject from "../../models/mudLog";
+import mudLog from "../../models/mudLog";
 import { ContentTable, ContentTableColumn, ContentTableRow, ContentType } from "./table";
 
-export interface MudLogObjectRow extends ContentTableRow {
-  message: MudLogObject;
+export interface mudLogRow extends ContentTableRow {
```

Comment:
Interface names should start with an uppercase letter

Reply:
Fixed